src/Entity/Course.php

```diff
@@ -101,6 +102,11 @@ public function __construct()
      */
     private $entryTask;
 
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Notification", mappedBy="course", cascade={"remove"})
+     */
+    private $notifications;
```

Comment:
O kodel geteriu / setteriu nera?

Reply:
Siaip as nelabai suprantu kodel tau cia is viso ju reikia. Turek omeny, kad kai is duombazes trauksi kursa, kartu su juo istrauksi ir koki 15 000 su juo susijusiu notificationu... :/